manifests/c/cmlanche/lightningvine/3.1.5+315/cmlanche.lightningvine.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: cmlanche.lightningvine
+PackageVersion: 3.1.5+315
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+ProductCode: 11370F74-8985-4099-A836-72497FE82A0F_is1
+Installers:
+- Architecture: x64
+  InstallerUrl: https://sdt-1314690352.cos.ap-nanjing.myqcloud.com/315/sdt-3.1.5-windows-installer.exe

```

Hi @SpecterShell,

The `PackageUrl` below took me to this download URL. Is there another webpage that has the `InstallerUrl` on it? 

```suggestion
  InstallerUrl: https://59-47-225-57.pd1.cjjd19.com:30443/download-cdn.123295.com/123-142/370e7274/1816849228-0/370e7274268331fc4e84570ffa6c488e/c-m96?v=5&t=1755132538&s=1755132538765b35be1c5f6ac0d1b63d4c32ce1f1a&r=RLK88N&bzc=2&bzs=313831363834393232383a323437303333383a33313939333537353a30&bzp=0&bi=653117240&filename=sdt-3.1.5-windows-installer.exe&x-mf-biz-cid=642d053b-683a-4d15-86b3-37018f12b0cc-c4937c&auto_redirect=0&ndcp=1&cache_type=1&xmfcid=0014c460-110c-47ab-b985-b7484402050b58a425c14b6
```